### PR TITLE
P0 create , like and dislike a post

### PR DESCRIPTION
**Pull request by dev-prabhat**
Implemented Creating, Like, and Dislike a post functionality  on the Home page 

- Link (https://github.com/dev-prabhat/nerd-community/pull/3/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&show-viewed-files=true)

### CODE REVIEW

**Review by pradipgharge — APPROVED** [review]
Code looks good and fine to merge.

File: src/features/post/postSlice.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import { createSlice , createAsyncThunk} from "@reduxjs/toolkit"
+import axios from "axios"
+
+const initialState = {
+    posts:[],
+    isLoading:false,
+    isRejected:false,
+    isLike:false,
+    isDisLike:false
```
Comment (mikesch803) [inline src/features/post/postSlice.js:9]: i guess you dont need states

**Review by mikesch803 — APPROVED** [review]
good to merge, nothing to suggest more

**Review by Dev-AkshatGupta — APPROVED** [review]
Everything looks good to me you can merge it

**Pull request merged**